golang/cosmos/x/vbank/module.go

```diff
@@ -188,14 +188,21 @@ func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.V
 	// Dump all the addressToBalances entries to SwingSet.
 	action := getBalanceUpdate(ctx, am.keeper, addressToBalance)
 	if action != nil {
-		am.PushAction(ctx, action)
+		err := am.PushAction(ctx, action)
+		if err != nil {
+			panic(err)
```

Comment:
I think it's better to log here and continue, as we do at 181 and 204, but I'm going to invite @michaelfig for an opinion. Giving an LGTM in advance of whatever decision is made.

Reply:
It's fine with me.